.github/workflows/build.yml

```diff
@@ -79,7 +79,7 @@ jobs:
 
   smoke-test:
     runs-on: ["zeta-runners"] # 25 minutes
-    timeout-minutes: 25
+    timeout-minutes: 30
```

Comment:
I think we might really need to consider making the smoke test more optimized or separating the smoke test into different types